models/Actor.php

```diff
@@ -78,7 +78,7 @@ public function describe(): string
     public function matches(Actor $actor): bool
     {
         return
-            ($actor->key !== null || $actor->id !== null || $actor->signkeys !== []) && // Match at least one of these
```

Comment:
You can also match on actor->identity

Reply:
Fixed in https://github.com/legalthings/workflow-engine/pull/1/commits/4a127a1c968a239086352912d9c7f58973fd8614